sdk/python/feast/infra/online_stores/redis.py

```diff
@@ -61,7 +61,7 @@ class RedisOnlineStoreConfig(FeastConfigBaseModel):
     type: Literal["redis"] = "redis"
     """Online store type selector"""
 
-    redis_type: RedisType = RedisType.redis
+    redis_type: Optional[RedisType] = RedisType.redis
```

Comment:
any reason for this change?

Reply:
I was testing required/optional parameters. I meant to take it back before commit